tap/tlstapper/tls_tapper.go

```diff
@@ -0,0 +1,174 @@
+package tlstapper
+
+import (
+	"bytes"
+	"encoding/binary"
+
+	"github.com/cilium/ebpf/perf"
+	"github.com/cilium/ebpf/rlimit"
+	"github.com/go-errors/errors"
+	"github.com/up9inc/mizu/shared/logger"
+)
+
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go tlsTapper bpf/tls_tapper.c -- -O2 -g -D__TARGET_ARCH_x86
+
+type TlsTapper struct {
+	bpfObjects   tlsTapperObjects
+	syscallHooks syscallHooks
+	sslHooks     []sslHooks
```

Comment:
We have a single set of syscallHooks because its the same for all processes.
We have multiple sets of sslHooks because it might be different per process. (various libssl.so running concurrently inside their container)

I can call it `perProcessSshHooks` although its not accurate as well, maybe `perLibrarySslHooks` which doesn't make sense as well

Reply:
Even `hookStructs` or `hookLists` will do.

Reply:
`sslHooksStucts`